notebooks/05b Modelling - regression fairness.ipynb

```diff
@@ -147,10 +164,20 @@
    },
    "outputs": [],
    "source": [
-    "# load models from outside the git tree\n",
-    "with open(\"../../models/regression.pickle\", \"rb\") as handle:\n",
-    "    models = pickle.load(handle)\n",
-    "models"
+    "for feature in sensitive_columns:\n",
+    "    data = pd.DataFrame(columns=[\"subfeature\", \"count\"])\n",
+    "    # iterate over all subcategories for the feature\n",
+    "    for subfeature in fairness_df[feature].unique():\n",
+    "        subset = fairness_df[fairness_df[feature] == subfeature]\n",
```

Comment:
fairness_df is referenced here before it has been created

Reply:
Apologies, this has been fixed